sdk/cosmos/azure-cosmos-spark_3_2-12/docs/configuration-reference.md

```diff
@@ -24,6 +24,7 @@
 | Config Property Name                                              | Default | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 |:------------------------------------------------------------------|:--------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| 
 | `spark.cosmos.useGatewayMode`                                     | `false` | Use gateway mode for the client operations                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `spark.cosmos.enforceNativeTransport`                             | `false` | Indicates whether native netty transport should be enforced. If true, on clinet creation an exception will be trown if netty is not able to use native transport. The native transport is more efficient especiall when there is a high number of connections.                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
wonder when this config will be needed

if the native library exists, then why customer would want to turn it off.  And if they want to enforce it and the native library does not exists, then it means we have a bug in our library as shading?